test/test_utils/__init__.py

```diff
@@ -148,6 +148,35 @@ def request_mxnet_inference_squeezenet(ip_address="127.0.0.1", port="80", connec
     return True
 
 
+@retry(
+    stop_max_attempt_number=10,
+    wait_fixed=10000,
+    retry_on_result=retry_if_result_is_false,
+)
+def request_mxnet_inference_resnet(ip_address="127.0.0.1", port="80", connection=None):
```

Comment:
This is almost exactly the same function as `request_mxnet_inference_squeezenet`. Could you merge these two functions?